velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -516,6 +516,25 @@ TEST_F(E2EFilterTest, largeMetadata) {
   EXPECT_EQ(1000, reader->numberOfRows());
 }
 
+TEST_F(E2EFilterTest, date) {
```

Comment:
Can we copy the same test on DWRF E2E test as well?  Since it's format independent

Reply:
DWRF does not support Date type right?

Reply:
ORC has it so it probably should be supported.  Just try copy the test there and see if it works, if it is not working then forget about it.

Reply:
@Yuhta I see the following error
```
unknown file: Failure
C++ exception with description "Exception: VeloxException
Error Source: RUNTIME
Error Code: UNKNOWN
Reason: not supported yet DATE
Retriable: False
Function: create
File: /Users/deepak/workspace/ahana/velox/velox/dwio/dwrf/writer/ColumnWriter.cpp
Line: 2058
```